### PR TITLE
Add smoke test to build pipeline

### PR DESCRIPTION
**Pull request by felixtheflex**
### Context

This is to add a smoke-test placeholder for Register app

### Changes proposed in this pull request

Add smoke-test.yml file, which is triggered by dispatch workflow, waits for result before testing the next deployment.

### CODE REVIEW

**Comment by felixtheflex** [comment]
[Add Smoke Test Register CI\CD workflow](https://trello.com/c/y3Qaen53/523-add-smoke-test-register-ci-cd-workflow)

**Review by saliceti — APPROVED** [review]

File: .github/workflows/smoke-test.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        run: echo "Nothing to run yet 🚀"
+
+      - name: 'Nofiy #twd_publish_register_tech on failure'
+        if: failure()
+        uses: rtCamp/action-slack-notify@master
+        env:
+          SLACK_CHANNEL: twd_publish_register_tech
+          SLACK_COLOR: '#ef5343'
+          SLACK_ICON_EMOJI: ':github-logo:'
+          SLACK_USERNAME: Teacher Training API
```
Comment (vigneshmsft) [inline .github/workflows/smoke-test.yml:29]:
```suggestion
          SLACK_USERNAME: Register Trainee Teachers
```

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -88,8 +87,26 @@ jobs:
           TF_VAR_dockerhub_username: ${{ secrets.DOCKERHUB_USERNAME }}
           TF_VAR_dockerhub_password: ${{ secrets.DOCKERHUB_PERSONAL_ACCESS_TOKEN }}
 
+      - name: Trigger ${{ env.DEPLOY_ENV }} Smoke Tests
+
```
Comment (vigneshmsft) [inline .github/workflows/deploy.yml:91]:
```suggestion
```

**Review by vigneshmsft — COMMENTED** [review]
also pls remove the london.cloudapps file

**Review by vigneshmsft — APPROVED** [review]
🚀

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (felixtheflex) [reply .github/workflows/deploy.yml]: done

**Pull request merged**